redbot/cogs/mod/kickban.py

```diff
@@ -101,14 +101,19 @@ async def ban_user(
         try:
             await guild.ban(user, reason=audit_reason, delete_message_days=days)
             log.info(
-                "{}({}) banned {}({}), deleting {} days worth of messages".format(
+                "{}({}) banned {}({}), deleting {} days worth of messages.".format(
                     author.name, author.id, user.name, user.id, str(days)
                 )
             )
         except discord.Forbidden:
             return _("I'm not allowed to do that.")
         except Exception as e:
-            return e  # TODO: improper return type? Is this intended to be re-raised?
+            log.exception(
+                "{}({}) attempted to kick {}({}), but an error occurred.".format(
+                    author.name, author.id, user.name, user.id
+                )
+            )
+            return _("An unexpected error occurred. ({})".format(e))
```

Comment:
String formatting needs to be done *after* calling `_()`
```suggestion
            return _("An unexpected error occurred. ({})").format(e)
```